app/build.gradle

```diff
@@ -12,7 +12,7 @@ android {
 
     defaultConfig {
         applicationId "com.onboarding.parkingsystemjava"
-        minSdkVersion 23
+        minSdkVersion 26
```

Comment:
undo this change

Reply:
I can't, one of the methods (can't remember which one) requires to have a minSdkversion set to 26

Reply:
![image](https://user-images.githubusercontent.com/84337198/122456024-6f4e4d00-cf83-11eb-9557-50b0cb9e9dbc.png)

Reply:
Facu, clearly you are doing something wrong, in a project, you can't change the min Sdk because you want

Reply:
fixed